specification/glossary.md

```diff
@@ -0,0 +1,44 @@
+# Glossary
+
+Below are a list of some OpenTelemetry specific terms that are used across this
+specification.
+
+## (Telemetry) Library
+
+Denotes the actual *OpenTelemetry* library in use.  
```

Comment:
Is this meant to be the API or SDK?

Reply:
It is my feeling that this is the SDK. The API should be the same for all vendors, but the SDK which is plugged will be different.

Reply:
Could be a fork of the API as well, or a compatible alternative to the standard API.

Reply:
If you fork the API, you will lose the main (planned) advantage of opentelemetry which is that modules will use the API to instrument themselves. Any module which uses the official version of the API will not report data to your forked version.

Reply:
Those questions regarding API vs SDK should be answered in the part of the specification I linked too.
@arminru already created an issue (https://github.com/open-telemetry/opentelemetry-specification/issues/484) for clearing that up.